### DISCUSSION TITLE
Temporarily disable 'auto-replace'

### DISCUSSION THREAD

**Discussion by Sejmou** [artisticat1/obsidian-latex-suite#194] [Q&A] [answered]
Hi,
first of all: I love this plugin - it saves soo much time writing LaTeX in Obsidian 😍 

However, there's one quite annoying issue that I haven't found a way around yet: sometimes things are 'auto-replaced' when I really don't want them to. For instance, when I try to write

```latex
\cdot
```

it gets replaced with
```latex
\\dot{c}
```

which is not what I want in this case. Is there any built-in way to temporarily tell the extension: "I know what I want to write, please just don't do anything at this moment!"?

Best regards,
Samo

**Comment by thudjx** [comment]
Hi Samo,

I got a similar issue as yours. Regarding your question, I found that when using the latest default snippet `cdot` will be replaced with `\cdot` as expected; the same for `ddot`. I have checked the default snippet and found the only magic is the "priority" settings in `dot` related snippet. 

What I am confusing is that when I want to write my own snippet for `ldots`, it doesn't work as expected. So I suspect that `cdot` works as I thought.

Nevertheless, you might want to go for a check for the default snippet to get `cdot` work as you wished.

**Comment by hieunguyen1212** [comment, marked as answer]
You can use a regex with lookaheads syntax to prevent `dot` from being triggered when it comes after a backslash. The trigger of the snippet should be `/(?<!\\\w*)dot/`.

**Comment by superle3** [comment]
> "I know what I want to write, please just don't do anything at this moment!"?

not really, you just have to check in the snippet itself, but due to ios 14.5 not supporting negative lookbehind and not wanting to complicate the default snippets with a bunch of functions returning false and doing some regex black magic, its now a bit incomplete. I'll try to track down the easier ones or just add `\b` to the beginning. 

like you can turn off the snippets with a hotkey, but doubt that is what you want.
